code/messaging/src/androidTest/java/com/adobe/marketing/mobile/messaging/MessagingPublicAPITests.java

```diff
@@ -77,7 +79,10 @@ public void setup() throws Exception {
         final CountDownLatch latch = new CountDownLatch(1);
         MobileCore.start((AdobeCallback) o -> latch.countDown());
 
-        latch.await(2, TimeUnit.SECONDS);
+        // wait for the initial edge personalization request to be made before resetting the monitor extension
+        List<Event> dispatchedEvents = getDispatchedEventsWith(MessagingTestConstants.EventType.EDGE,
```

Comment:
When is the initial Edge request made? Is it arising from any event sent from the Messaging extension?

Reply:
the messaging extension makes an initial edge personalization request once the configuration and identity is set: https://github.com/adobe/aepsdk-messaging-android/blob/feature/feed/code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/MessagingExtension.java#L204-L207